src/tech/how-we-do-development.md

```diff
@@ -163,9 +163,8 @@ origin [BRANCHNAME]`.
 
 ### 8. Submit the changeset for code review
 
-At dxw, we never make changes to code running in production without the involvement of at least two people. In practice, this means one author and at least one approving reviewer, or authoring code as a pair or mob. This to make
-sure we're delivering high quality software and it's an opportunity to learn
-from each other.
+At dxw, we never make changes to code running in production without the involvement of at least two people. In practice, this means one author and at least one approving reviewer, or authoring code as a pair or mob. We do this to make
+sure we're delivering high quality software and it's an opportunity to learn from each other.
```

Comment:
Maybe `...and use it as an opportunity...`?

Reply:
Yes i agree, that reads better - done